### PR TITLE
add tron transfers

### PR DESCRIPTION
**Pull request by 0xRobin**
add tron transfers.

blocked by https://github.com/duneanalytics/spellbook/pull/6995

### CODE REVIEW

**Comment by 0xBoxer** [comment]
Due to a dbt version change, you need to merge main into your branch before the Github action runner runs successfully again.

**Comment by 0xRobin** [comment]
@jeff-dude I'm marking blocked as we'll need to add an adjustment to the tokens metadata join when we figure out the decoding

File: dbt_subprojects/tokens/models/transfers_and_balances/tron/tokens_tron_base_transfers.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    , t.value AS amount_raw
+FROM {{ source('erc20_tron','evt_transfer') }} t
+INNER JOIN {{ source('tron','transactions') }} tx ON
+    tx.block_time = t.evt_block_time -- raw base tables partitioned by time
+    AND tx.block_number = t.evt_block_number
+    AND tx.hash = t.evt_tx_hash
+    {% if is_incremental() or true %}
+    AND {{incremental_predicate('tx.block_time')}}
+    {% endif %}
+{% if is_incremental() or true %}
```
Comment (jeff-dude) [inline dbt_subprojects/tokens/models/transfers_and_balances/tron/tokens_tron_base_transfers.sql:40]:
```suggestion
{% if is_incremental() %}
```

Comment (jeff-dude) [reply dbt_subprojects/tokens/models/transfers_and_balances/tron/tokens_tron_base_transfers.sql:40]: assuming was to get CI to pass, as the history + incremental had same row counts (love that feature in latest version)

**Review by jeff-dude — APPROVED** [review]
approving with final cleanup suggestion below

note: these transfers spells can take a long time in prod, if data volume on tron is any bit similar to bnb/polygon/ethereum. we can chat as needed to prep the env

**Comment by 0xRobin** [comment]
when merging, we need to make sure `tokens.erc20` is ran first, before the transfer models are run.

**Pull request merged**